install/Knative-with-OpenShift.md

```diff
@@ -11,168 +11,154 @@ You can find [guides for other platforms here](README.md).
-local machine using [`oc cluster up`](https://docs.openshift.org/latest/getting_started/administrators.html#running-in-a-docker-container)
-to test-drive knative.
+local machine using [minishift](https://docs.okd.io/latest/minishift/getting-started/index.html)
+to test-drive Knative.
 
-## Install `oc` (openshift cli)
+## Configure and start minishift
 
-You can install the latest version of `oc`, the OpenShift CLI, into your local
-directory by downloading the right release tarball for your OS from the
-[releases page](https://github.com/openshift/origin/releases/tag/v3.10.0).
+The following details the bare minimum configuration required to setup minishift fro running Knative:
 
 ```shell
-export OS=<your OS here>
-curl https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-$OS-64bit.tar.gz -o oc.tar.gz
-tar zvf oc.tar.gz -x openshift-origin-client-tools-v3.10.0-dd10d17-$OS-64bit/oc --strip=1
-
-# You will now have the oc binary in your local directory
+minishift profile set knative
+minishift config set memory 8GB
+minishift config set cpus 4
+minishift config set image-caching true
+minishift addon enable admin-user
+minishift addon enable anyuid
 ```
 
-## Scripted cluster setup and installation
-
-For Linux and Mac, you can optionally run a
-[script](scripts/knative-with-openshift.sh) that automates the steps on this
-page.
+The above configuration ensures that Knative gets created in its own [minishift profile](https://docs.okd.io/latest/minishift/using/profiles.html) with 8GB of RAM and 4 vCpus. The image-caching helps in re-starting up the cluster faster every time.  The [addon](https://docs.okd.io/latest/minishift/using/addons.html) **admin-user** creates a default `admin` user with the role  cluster-admin and the  [addon](https://docs.okd.io/latest/minishift/using/addons.html) **anyuid** allows the `default` service account to run the application with uid `0`.
 
-Once you have `oc` present on your machine and in your `PATH`, you can simply
-run [this script](scripts/knative-with-openshift.sh); it will:
+To start minishift:
 
-- Create a new OpenShift cluster on your local machine with `oc cluster up`
-- Install Istio and Knative serving
-- Log you in as the cluster administrator
-- Set up the default namespace for istio autoinjection
+```shell
+minishift profile set knative
+minishift start
+```
 
-Once the script completes, you'll be ready to test out Knative!
+The command `minishift profile set knative` is required every time you start and stop minishift to make sure that you are on right `knative` minishift profile that was configured above.
 
-## Creating a new OpenShift cluster
+## Configuring `oc` (openshift cli)
 
-Create a new OpenShift cluster on your local machine using `oc cluster up`:
+Running the following command make sure that we have right version of `oc` and configured the DOCKER daemon to be connected to minishift Docker.
```

Comment:
"and configured" --> "and have configured"

Reply:
done